Documentation/csiaddons.md

```diff
@@ -0,0 +1,93 @@
+# CSI-Addons for Kubernetes
+
+## Table of Contents
+
+- [CSI-Addons for Kubernetes](#csi-addons-for-kubernetes)
+  - [Table of Contents](#table-of-contents)
+  - [CSI-Addons Controller](#csi-addons-controller)
+    - [Installation by operator](#installation-by-operator)
+  - [Enable the CSI-Addons Sidecar](#enable-the-csi-addons-sidecar)
+  - [Deploying the CSI-Addons Controller](#deploying-the-csi-addons-controller)
+
+## CSI-Addons Controller
+
+The CSI-Addons Controller handles the requests from users to initiate an operation. Users create a CR that the controller inspects, and forwards a request to one or more CSI-Addons side-cars for execution.
+By listing the CSIAddonsNode CRs, the CSI-Addons Controller knows how to connect to the side-cars. By checking the supported capabilities of the side-cars, it can decide where to execute operations that the user requested.
+
+### Installation by operator
+
+A CSI-Addons bundle can be used to install the CSI-Addons Controller with the following steps:
+
+```bash
```

Comment:
`make operator-sdk` is specific to the kubernetes-csi-addons repository. It does not matter how, or where the user gets the `operator-sdk` executable from.

```bash
$ kubectl create namespace storage-csi-addons
$ operator-sdk run bundle -n storage-csi-addons quay.io/csiaddons/k8s-bundle:latest
```

should be sufficient?